yew/src/virtual_dom/vlist.rs

```diff
@@ -34,6 +41,22 @@ impl DerefMut for VList {
     }
 }
 
+#[cfg(feature = "sans_mount_render")]
+impl TryFrom<VList> for Html {
+    type Error = HtmlRenderError;
+
+    fn try_from(value: VList) -> Result<Html, HtmlRenderError> {
+        let mut result: String = "".to_string();
+        for child in value.children {
+            let html = Html::try_from(child)?.to_string();
+            result.push_str(html.as_ref());
+        }
+
+        result.shrink_to_fit();
```

Comment:
Not sure why it should belong in `Html::new`; that function doesn't know where the string came from or what kind of state it might be in. I would prefer to keep memory cleanup at the site of the place that may have caused any memory usage issues to begin with.

Reply:
My argument is that the `Html` struct knows that the `String` it accepts no longer needs to grow so it can safely shrink it.
Is there a situation where you wouldn't want to call `shrink_to_fit` before passing it to `Html`? If there isn't then I don't see why we wouldn't move it there for simplicity.

I'm fine with leaving it as is though.